image/base/slim.Dockerfile

```diff
@@ -1,13 +1,18 @@
 # NAME:     discourse/base
 # VERSION:  release
-FROM debian:bullseye-slim
 
+ARG DEBIAN_RELEASE=bullseye
+FROM debian:${DEBIAN_RELEASE}-slim
+
+ARG DEBIAN_RELEASE
 ENV PG_MAJOR=13 \
     RUBY_ALLOCATOR=/usr/lib/libjemalloc.so \
     RUSTUP_HOME=/usr/local/rustup \
     CARGO_HOME=/usr/local/cargo \
     PATH=/usr/local/cargo/bin:$PATH \
-    LEFTHOOK=0
+    LEFTHOOK=0 \
+    RUBY_VERSION=3.2.3 \
+    DEBIAN_RELEASE=${DEBIAN_RELEASE}
```

Comment:
Initially I wanted to set `DEBIAN_VERSION` based on the output from `cat /etc/debian_version` but there isn't a way to do so.